### ISSUE TITLE
Include OpenAPI 1.1

### ISSUE THREAD

**Issue by kwsutter** [eclipse/microprofile#68]
As OpenAPI 1.1 makes progress, update the MicroProfile pom.xml file with the appropriate dependency information.  This issue will be used to track the progress of including OpenAPI 1.1.

**Comment by jmini** [comment]
Please notice that due to some issues in the TCK with version `1.1`, a new version [`1.1.1`](https://github.com/eclipse/microprofile-open-api/releases/tag/mp-openapi-1.1.1) was published. This is the version that should be used in the next MP release

**Comment by kwsutter** [comment]
@jmini Thanks, Jeremie!  Once the 1.1.1 release is published, then I would ask you to create a PR for the pom.xml for including it in the MP 2.2 release.  Thanks!

**Comment by jmini** [comment]
The release is published, corresponding PR: https://github.com/eclipse/microprofile/pull/80

**Comment by arthurdm** [comment]
Thanks @jmini!

**Comment by Emily-Jiang** [comment]
Based on our policy, we don't include bug releases. Basically, the umbrella spec only include 1.1 release as bug release will be ignored. However the implementation can certify against the 1.1.1 version of tck.

**Comment by kwsutter** [comment]
@Emily-Jiang Maybe that's a convention, but I can't find where it's a "policy".  TBH, since our third digit patch releases are only used for TCK updates, then specifying OpenAPI 1.1 or 1.1.1 really doesn't matter from an API perspective and that's what this pom.xml is for.  And, since OpenAPI 1.1.1 already exists and is released, shouldn't we include that in the pom?

We already break the "rule" of not specifying a third digit on the JAX-RS 2.0.1 dependency (Java EE 7)...  Maybe this is another versioning clarification discussion that needs to happen?  I know @jclingan is working on a "platform document", maybe it should be part of that discussion when we review it?

@Emily-Jiang, is there a potential problem that you are foreseeing with including OpenAPI 1.1.1?  Thanks!

**Comment by Emily-Jiang** [comment]
@kwsutter This policy was agreed during our live [hangout](https://docs.google.com/document/d/16v3jVkcDzVz9BVU5aGEzPVbK-a8BIx7S1gbqToVUaLs/edit) Dec 2017 that we will not include micro/bug release into umbrella releases. The reason is that we don't track bug fixes and there is not much impact to end users as there are no api changes. In this way, it will make umbrella release easier to manage and bug releases can be done outside of the release cycle and can be done any number of times.

**Comment by Emily-Jiang** [comment]
@arthurdm please change to include open api 1.1 only. In other specs, we have done a number of bug fixes and none of them were included under umbrella releases. In this way, there is no differentiation among the micro/bug releases.

**Comment by jmini** [comment]
@Emily-Jiang : this is very unfortunate... The `1.1` was released too quickly and contains issue in the TCK and should not be used.

This is why a `1.1.1` was created after @EricWittmann fixes the `1.1` TCK issues. (`1.1.1` is basically a re-spin of `1.1`).

@Emily-Jiang:
What is your proposal to move forward?

**Comment by EricWittmann** [comment]
Just to add a little bit of clarity to the scope of the problem with OpenAPI 1.1 - the 1.1 version of the MP+OAI TCK has a bug that makes it **impossible** for an implementation to pass.  Just so we're all clear on this point.  :)

We didn't make any specification changes between 1.1 and 1.1.1, so I don't think there is any impact on applications using the spec.

**Comment by Emily-Jiang** [comment]
@EricWittmann It is ok to release 1.1.1 but we are not tracking bug fixes. Your github has the releases listed. The impls can pick up the latest release to pass.
By the way, a few specs have hit the same problem before. There are no impacts for end users but for implementors to certify.

**Comment by kwsutter** [comment]
Hi @EricWittmann and @jmini...  I talked with  @Emily-Jiang  and she is correct.  We did decide that only the major.minor version of a spec should be specified in the Platform delivery.  The patch version (major.minor.patch) is only used for TCK updates per MicroProfile conventions.  Thus, the Platform pom.xml should only reference the OpenAPI 1.1 release.  The latest TCK (1.1.1) will need to executed by any implementations of OpenAPI.  Make sense?

Bottom line is that we need an updated PR to list the OpenAPI 1.1 dependency instead of 1.1.1.  Thanks!

**Comment by kwsutter** [comment]
PR #86 merged!
